src/util/buildScript.2017071.js

```diff
@@ -62,6 +72,7 @@ function buildNewRxPatient(doc, patientResource) {
 function buildNewRxPrescriber(doc, practitionerResource) {
   var prescriber = doc.createElement('Prescriber');
   var nonVeterinarian = doc.createElement('NonVeterinarian');
+  var npi = null;
 
   //     Prescriber Identifier
   for (let i = 0; i < practitionerResource.identifier.length; i++) {
```

Comment:
```suggestion
  if (npi) {
    var identification = doc.createElement('Identification');
    xmlAddTextNode(doc, identification, 'NPI', npi);
    nonVeterinarian.appendChild(identification);
  }
```
meant to replace for loop (GitHub won't allow suggesting on lines 79-80)